doc/source/release-notes/Percona-XtraDB-Cluster-5.7.20-29.24.rst

```diff
@@ -0,0 +1,121 @@
+.. rn:: 5.7.20-29.24
+
+=====================================
+Percona XtraDB Cluster 5.7.20-29.24
+=====================================
+
+Percona is glad to announce the release of
+Percona XtraDB Cluster 5.7.20-29.24 on January 26, 2018.
+Binaries are available from the `downloads section
+<http://www.percona.com/downloads/Percona-XtraDB-Cluster-57/>`_
+or from our :ref:`software repositories <install>`.
+
+.. note:: Due to new package dependency,
+   Ubuntu/Debian users should use ``apt-get dist-upgrade``, ``apt upgrade``,
+   or ``apt-get install percona-xtradb-cluster-57`` to upgrade.
+
+Percona XtraDB Cluster 5.7.20-29.24 is now the current release,
+based on the following:
+
+* `Percona Server 5.7.20-18 <https://www.percona.com/doc/percona-server/5.7/release-notes/Percona-Server-5.7.20-18.html>`_
+
+* Galera Replication library 3.22
+
+* Galera/Codership WSREP API Release 5.7.20
+
+All Percona software is open-source and free.
+
+NEW FEATURES:
+=============
+
+* Ubuntu 17.10 Artful Aardvark is now supported.
+
+* Freezing gcache purge was implemented to facilitate node
+  joining through IST, avoiding time consuming SST process.
+
+* :jirabug:`PXC-822`: an usability improvement was made in timeout error
+  messages making them to name the configuration variable which caused this
+  timeout.
+
+* :jirabug:`PXC-868`: on the Joiner, during SST, "tmpdir" variable under
+  [sst] section can be used to specify temporary SST files storage different
+  from the default datadir/.sst one.
+
+* A new variable :variable:`wsrep_last_applied` in addition
+  to :variable:`wsrep_last_committed` one was introduced to clearly separate
+  last committed and last applied transaction numbers.
```

Comment:
added commas

----
A new variable :variable:`wsrep_last_applied`, in addition
to :variable:`wsrep_last_committed`, one was introduced to clearly separate
last committed and last applied transaction numbers.